### DISCUSSION TITLE
package 'tailwindcss' not found

### DISCUSSION THREAD

**Discussion by FoldZero** [tailwindlabs/tailwindcss#15334] [Help]
the tailwind.csss package is missing from https://www.npmjs.com/package/tailwindcss

![image](https://github.com/user-attachments/assets/8fdc8b16-17cb-414c-a287-a541b9ddee24)

**Comment by wongjn** [comment]
This is probably due to [NPM's infrastructure upgrade](https://status.npmjs.org/incidents/px35z3dmcjgm).

**Reply by FoldZero** [reply]
> This is probably due to [NPM's infrastructure upgrade](https://status.npmjs.org/incidents/px35z3dmcjgm).

cool, its back now
